src/renderer/services/db/api/repositories/attestations.ts

```diff
@@ -0,0 +1,12 @@
+import {BulkRepository} from "../repository";
+import {IDatabaseController} from "../../../../../main/db/controller";
+import {Bucket} from "../../schema";
+import {JSONSerializer} from "../../serializers/json";
+import {types as mainnetTypes} from "@chainsafe/eth2.0-types/lib/ssz/presets/mainnet";
+import {Attestation} from "@chainsafe/eth2.0-types";
+
+export class AttestationsRepository extends BulkRepository<Attestation> {
+    public constructor(db: IDatabaseController) {
+        super(db, JSONSerializer, Bucket.attestations, mainnetTypes.Attestation);
+    }
```

Comment:
In my opinio it's better to extend every method here to accept validator pubkey. It makes api a lot more friendlier